pegasus/sites.v3/code.org/views/visit_partial_event_logger.erb

```diff
@@ -0,0 +1,23 @@
+<%
```

Comment:
this looks great for erb, but I think it will be worth converting to haml which is more standard for us. you can just ues a single `:ruby` block to wrap all of the code in this erb block:
```suggestion
:ruby
```

Reply:
Sounds good!